app/services/forest_liana/search_query_builder.rb

```diff
@@ -79,16 +79,18 @@ def search_param
           end
         end
 
-        SchemaUtils.one_associations(@resource).map(&:name).each do |association|
-          if @includes.include? association.to_sym
-            resource = @resource.reflect_on_association(association.to_sym)
-            resource.klass.columns.each do |column|
-              if !(column.respond_to?(:array) && column.array) &&
-                (column.type == :string || column.type == :text)
-                column_name = format_column_name(resource.table_name,
-                  column.name)
-                conditions << "LOWER(#{column_name}) LIKE " +
-                  "'%#{@params[:search].downcase}%'"
+        if (@params['deep'].to_i == 1)
```

Comment:
Actually, not a fan of this parameter name. "deep" is not that explicit. `searchType` (`simple` vs `deep`) would be more explicit.